lib/resources/trending_movies_api_provider.dart

```diff
@@ -0,0 +1,27 @@
+import 'dart:convert';
+import 'package:http/http.dart' as http;
+import '../constants/constants.dart';
+import '../models/item_model.dart';
+
+class TrendingMoviesApiProvider {
+  final String _endpoint = "/trending/movie/week";
```

Comment:
You can add this variable into Constant file.

Reply:
Done!